### PR TITLE
Paypal Subscription added

### PR DESCRIPTION
**Pull request by tejassonar**
Paypal subscription 

@

### CODE REVIEW

File: src/Donations/PaymentMethods/NewPaypal.tsx
Comment (norbertschuler) [inline src/Donations/PaymentMethods/NewPaypal.tsx:79]: Don't forget to remove this before merging:

Bildschirmfoto 2021-12-27 um 13 33 04

**Review by norbertschuler — DISMISSED** [review]
The Paypal webhook for my donation https://app-staging.plant-for-the-planet.org/backend/donationContribution/572799/show failed with a 500 error at https://sentry.io/organizations/plant-for-the-planet/issues/2888937126/?project=1293762&referrer=slack

File: src/Donations/PaymentMethods/NewPaypal.tsx
Comment (tejassonar) [reply src/Donations/PaymentMethods/NewPaypal.tsx]: Yes. I removed an alert.

File: src/Donations/PaymentMethods/NewPaypal.tsx
Comment (norbertschuler) [inline src/Donations/PaymentMethods/NewPaypal.tsx:88]: Let's remove this before merging.

**Review by norbertschuler — COMMENTED** [review]
Frontend code looks good for me. Also the subscription seem to be added correctly in the Paypal sandbox. Just the Paypal web hook still failed with 500 error at Sentry https://sentry.io/organizations/plant-for-the-planet/issues/2903593084/?project=1293762&referrer=slack

**Review by norbertschuler — APPROVED** [review]
Approving this to get merged once the backend is fixed.

File: src/Donations/PaymentMethods/NewPaypal.tsx
Comment (sagararyal) [inline src/Donations/PaymentMethods/NewPaypal.tsx:87]: Translate.

Comment (norbertschuler) [reply src/Donations/PaymentMethods/NewPaypal.tsx]: Come on - please do not just fix this, fix them all! :-)

Bildschirmfoto 2022-01-04 um 14 06 29

**Comment by norbertschuler** [comment]
> Approving this to get merged once the backend is fixed.

Backend on staging seems to work now. Before making a Frontend release we of course need a backend release with the fix.

**Review by norbertschuler — APPROVED** [review]
LGFM

**Comment by norbertschuler** [comment]
Before releasing this PR we need to adapt/restrict the possible operations on recurring donations in the planet webapp according to the information by the backend team as otherwise some operations on Paypal subscription would fail as e.g. described in https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/1407 or https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/1406:

image (9)

**Comment by mariahosfeld** [comment]
Closing this.

**Pull request closed without merging**
